addons/google_calendar/models/google_sync.py

```diff
@@ -235,10 +235,10 @@ def _google_error_handling(self, http_error):
                 )
 
     @after_commit
-    def _google_delete(self, google_service: GoogleCalendarService, google_id, timeout=TIMEOUT):
+    def _google_delete(self, google_service: GoogleCalendarService, google_id, is_recurrence=False, timeout=TIMEOUT):
```

Comment:
Unfortunately, we can't change the method signature in stable because if some client override it, the code will crash.
https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean